pulsar-common/src/main/java/org/apache/pulsar/common/api/proto/PulsarApi.java

```diff
@@ -270,13 +270,13 @@ private ProtocolVersion(int index, int value) {
       com.google.protobuf.GeneratedMessageLite
       implements MessageIdDataOrBuilder, org.apache.pulsar.common.util.protobuf.ByteBufCodedOutputStream.ByteBufGeneratedMessage  {
     // Use MessageIdData.newBuilder() to construct.
-    private final io.netty.util.Recycler.Handle<MessageIdData> handle;
-    private MessageIdData(io.netty.util.Recycler.Handle<MessageIdData> handle) {
+    private io.netty.util.Recycler.Handle handle;
```

Comment:
We have updated the patched `protoc` to use the generic `Recycler` that was introduced in Netty-4.1. Can you regenerate the code from the latest build image?

```shell
docker pull apachepulsar/pulsar-build:latest
cd ${PULSAR_HOME}/pulsar-common/
./generate_protobuf_docker.sh
```

Reply:
will do